vscode-wpilib/resources/media/main.js

```diff
@@ -202,13 +271,32 @@
         vscode.postMessage({ type: 'updateall' });
       });
 
+    document
+      .getElementById('install-url-action')
+      ?.addEventListener('click', () => {
+        const urlInput = /** @type {HTMLInputElement | null} */ (document.getElementById('url-input'));
+        if (urlInput && urlInput.value.trim()) {
+          vscode.postMessage({ 
+            type: 'installFromUrl', 
+            url: urlInput.value.trim() 
+          });
+          urlInput.value = '';
+        }
+      });
+
+    document
+      .getElementById('url-input')
+      ?.addEventListener('keypress', (event) => {
+        if (event.key === 'Enter') {
+          document.getElementById('install-url-action')?.click();
```

Comment:
Nit: do we really want to trigger like this? It would be nicer IMO to call manually- sending click feels a little gross to me